.github/workflows/ci.yaml

```diff
@@ -46,4 +46,4 @@ jobs:
       - name: Lint 
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.29
+          version: v1.45
```

Comment:
Was there a driving reason for this or are we just staying up to date?

Reply:
Little bit of both -- locally I happen to have 1.45 and that's what I was using to lint. But I was having a hard time getting this to pass in CI without upgrading. I suspect it may be because we're using v2 instead of pinned to a specific 2.x.x version of the action, they may have dropped support for older versions of golangci-lint.